experimental/instrumentation-fetch/src/instrumentation.test.ts

```diff
@@ -36,7 +37,10 @@ describe('FetchInstrumentation', () => {
     const instrumentation = new FetchInstrumentation({
       testing: true,
     });
-    instrumentation.initialize();
+
+    const sessionId = 'test-session-id';
+
+    initializeFaro(mockConfig({ instrumentations: [instrumentation], session: { id: sessionId } }));
```

Comment:
Soon we will remove the old session handling and only use the new session management. This will cause this test to fail.

I think the best way is to spy on the `createSession()` function and use `mockReturnValueOnce` to return a meta with a  fixed sessionId.